docs/design.md

```diff
@@ -63,24 +63,26 @@ The meta.json is updated during upload time on sidecars.
               │                   Object Storage                 │
               └──────────────────────────────────────────────────┘
 ```
+#### Note
```

Comment:
So add definitions for chunk and chunk file and please link them accordingly.

Reply:
@kakkoyun Done !